bitrise.yml

```diff
@@ -34,25 +34,10 @@ workflows:
     steps:
     - deploy-to-bitrise-io: {}
 
-  run_linter:
+  run_check_tools:
     description: |-
-        Runs swiftlint and license check for all source and test files.
+        Runs internal tools to check the project.
     steps:
-    - swiftlint@0.8.0:
-        title: Lint Sources/*
-        inputs:
-        - strict: 'yes'
-        - lint_config_file: "$BITRISE_SOURCE_DIR/tools/lint/sources.swiftlint.yml"
-        - linting_path: "$BITRISE_SOURCE_DIR"
-        - reporter: emoji
-    - swiftlint@0.8.0:
-        title: Lint Tests/*
-        is_always_run: true
-        inputs:
-        - strict: 'yes'
-        - linting_path: "$BITRISE_SOURCE_DIR"
-        - lint_config_file: "$BITRISE_SOURCE_DIR/tools/lint/tests.swiftlint.yml"
-        - reporter: emoji
```

Comment:
Those steps were linting more than SDK code, see linter configuration:
```yml
# tools/lint/sources.swiftlint.yml:

  - instrumented-tests/http-server-mock/Sources
  - tools/api-surface/Sources
  - tools/api-surface/Fixtures/Sources
  - tools/rum-models-generator/Sources
  - tools/rum-models-generator/Tests/rum-models-generator-coreTests/Fixtures/Output # lint generation fixture
  - dependency-manager-tests/carthage/CTProject
  - dependency-manager-tests/cocoapods/CPProject
  - dependency-manager-tests/spm/SPMProject

# tools/lint/tests.swiftlint.yml:

  - instrumented-tests/http-server-mock/Tests
  - tools/api-surface/Tests
  - tools/rum-models-generator/Tests
  - dependency-manager-tests/carthage/CTProjectTests
  - dependency-manager-tests/carthage/CTProjectUITests
  - dependency-manager-tests/cocoapods/CTProjectTests
  - dependency-manager-tests/cocoapods/CTProjectUITests
  - dependency-manager-tests/spm/SPMProjectTests
  - dependency-manager-tests/spm/SPMProjectUITests
```

Now, these sources are lint as part of  _**"Run unit tests for Datadog - iOS Simulator"**_ which is wrong (we shouldn't fail SDK tests due to linter issue in `rum-models-generator` tool).